src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/UpdateSubmission.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission.changes.Change;
+
+import java.util.List;
+
+public class UpdateSubmission {
+    @JsonProperty("type")
+    private String type = "OE02";
+
+    @JsonProperty("userSubmission")
+    private OverseasEntitySubmissionDto userSubmission;
+
+    @JsonProperty("dueDiligence")
+    private DueDiligence dueDiligence;
+
+    @JsonProperty("presenter")
+    private Presenter presenter;
+
+    @JsonProperty("filingForDate")
+    private FilingForDate filingForDate;
+
+    @JsonProperty("noChangesInFilingPeriodStatement")
+    private String noChangesInFilingPeriodStatement;
+
+    @JsonProperty("anyBOsOrMOsAddedOrCeased")
+    private String anyBOsOrMOsAddedOrCeased;
+
+    @JsonProperty("beneficialOwnerStatement")
+    private String beneficialOwnerStatement;
+
+    @JsonProperty("changes")
+    private List<Change> changes;
+
+    @JsonProperty("additions")
+    private List<Change> additions;
```

Comment:
Need to work on a design how the additions will be stored -- perhaps not as a Change but Additions Type

Reply:
Here's what the target xml will look like for Managing Officers additions --
`<appointments>
      <officers>
        <officer>
          <personName>
            <titleOther>MR</titleOther>
            <surname>TESTER</surname>
            <forename>TESTY</forename>
            <middlenames>TEST</middlenames>
          </personName>
          <actionDate>2009-01-01</actionDate>
          <corporateOfficer>false</corporateOfficer>
          <otherNames />
          <serviceAddress>
            <isServiceAddressROA>true</isServiceAddressROA>
          </serviceAddress>
          <residentialAddress>
            <address>
              <houseNameNum>72</houseNameNum>
              <street>BELFORD ROAD</street>
              <area />
              <postTown>EDINBURGH</postTown>
              <country>SCOTLAND</country>
              <postCode>EH4 3DS</postCode>
              <addressValidation>valid</addressValidation>
            </address>
            <isResidentialAddressSA>false</isResidentialAddressSA>
          </residentialAddress>
          <nationalityOther>WELSH</nationalityOther>
          <birthDate>1987-09-08</birthDate>
          <businessOccupation>TESTER</businessOccupation>
          <usualResidence>WALES</usualResidence>
          <personalAuthentication>
            <authenticationStatus>1</authenticationStatus>
            <authenticationPair>
              <authenticationItem>03</authenticationItem>
              <authenticationData>931</authenticationData>
            </authenticationPair>
            <authenticationPair>
              <authenticationItem>01</authenticationItem>
              <authenticationData>HAV</authenticationData>
            </authenticationPair>
            <authenticationPair>
              <authenticationItem>05</authenticationItem>
              <authenticationData>40B</authenticationData>
            </authenticationPair>
          </personalAuthentication>
        </officer>
      </officers>
    </appointments>`

Reply:
`<appointments>
      <officers />
    </appointments>`

`<pscAppointments>
      <pscs>
        <psc>
        </psc>
      </pscs>
    </pscAppointments>`

Reply:
please take this out of this PR